src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
```

Comment:
- [ ]  Nice work so far in creating your initialState, but as per the project you are to have an empty categories array

```javascript
initialState: {
    categories: [],
  },
```